app/controllers/report_controller.rb

```diff
@@ -536,6 +536,7 @@ def get_node_info(_node = {}, _show_list = true)
     when "g"    then determine_g_node_info(nodeid)
     when "xx"   then determine_xx_node_info
     when "rr"   then determine_rr_node_info
+    when "ws"   then determine_xx_node_info
```

Comment:
so the XX is staying and now WS and XX does the same?

Reply:
Yeah, it has to be, because in some crazy cases we're still handling it as `xx` node. Because the `@nodetype` isn't the last node in the full_id, but something in the middle :confused: :nauseated_face: